jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/InvoiceResource.java

```diff
@@ -669,9 +669,7 @@ public Response createInstantPayment(@PathParam("invoiceId") final UUID invoiceI
                                          @javax.ws.rs.core.Context final HttpServletRequest request,
                                          @javax.ws.rs.core.Context final UriInfo uriInfo) throws AccountApiException, PaymentApiException {
         verifyNonNullOrEmpty(payment, "InvoicePaymentJson body should be specified");
-        verifyNonNullOrEmpty(payment.getAccountId(), "InvoicePaymentJson accountId needs to be set",
-                             payment.getTargetInvoiceId(), "InvoicePaymentJson targetInvoiceId needs to be set",
-                             payment.getPurchasedAmount(), "InvoicePaymentJson purchasedAmount needs to be set");
```

Comment:
I've double checked that passing `null` works as expected: https://github.com/killbill/killbill/blob/381ad0ebfceb5cd660f1fe9bcc05e842a250b0c2/payment/src/main/java/org/killbill/billing/payment/invoice/InvoicePaymentControlPluginApi.java#L677